hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/ProcessorTasklet.java

```diff
@@ -172,7 +170,7 @@ public ProcessorTasklet(
                                     .sorted(comparing(OutboundEdgeStream::ordinal))
                                     .toArray(OutboundEdgeStream[]::new);
         this.ssContext = ssContext;
-        this.logger = getLogger(context);
+        this.logger = new PrefixedLogger(context.logger(), toString());
```

Comment:
Previous code used `getLogger(getClass() + "." + toString())`. Now we use `context.logger()` which already contains vertex information, so we'll log the vertex information twice.